api/kubernetes/helm-chart/templates/deployment.yaml

```diff
@@ -75,6 +75,11 @@ spec:
               # The properties file for the 'helm' profile is mounted as a Volume in the Container,
               # using a configured Kubernetes Secret containing all the 'config.*' properties
               value: "helm"
+          {{ if .Values.config.secrets.from_env.enabled }}
```

Comment:
Could you please rename `from_env` in `envFrom` please?

Reply:
done